metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -202,8 +202,22 @@ case class ScalafixProvider(
       .getOrElse("Unexpected error while running Scalafix.")
   }
 
-  private def scalafixConf: Option[Path] = {
+  private lazy val scala3DefaultConfig = {
```

Comment:
Turns out that Scala organize imports will fail with an empty confix for Scala3, so I added a temporary config file.